sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2837,6 +2855,11 @@ export class StorageSharedKeyCredentialPolicy extends CredentialPolicy {
 // @public
 export type SyncCopyStatusType = 'success';
 
+// @public
+export interface TagConditions {
+    ifTags?: string;
```

Comment:
should this be `ifTagsMatch`? It makes more sense to me...

Reply:
.NET is using a `TagConditions` string; do we need to align with them?
```
   public class BlobLeaseRequestConditions : RequestConditions
    {
        /// <summary>
        /// Optional SQL statement to apply to the Tags of the Blob.
        /// </summary>
        public string TagConditions { get; set; }
    }
```

Reply:
I like `ifTagsMatch` because it aligns with `ifMatch` in semantics I think, except using tags instead of etag?

Otherwise, I like tagConditions better than ifTags personally.

Reply:
I don't like `ifTagsMatch`. `ifMatch` is basically `ifEtagMatch`. It will be used like this: ` ifMatch: "An ETag value"`.
While `ifTags` is a predicate, for example `Status = 'Done' or LastModified > '2018-06-18 20:51:26Z'`. Maybe it's better to be named `tagsPredicate` but let's follow .NET and go with `tagConditions`.

https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-conditional-headers-for-blob-service-operations#tags-conditional-operations

Conditional header | Description
-- | --
x-ms-if-tags | Version 2019-12-12 and newer. A TagsPredicate value. Specify this header to perform the operation only if the predicate evaluates to true against the blob's tags.
x-ms-source-if-tags | Version 2019-12-12 and newer. Applies to Copy Blob only. A TagsPredicate value. Specify this header to perform the operation only if the predicate evaluates to true against the source blob's tags.